### ISSUE TITLE
Tooltip appears at the wrong place

### ISSUE THREAD

**Issue by damithc** [MarkBind/markbind#12]
In the example below, the word `coupling` is the one that has a tooltip

![image](https://cloud.githubusercontent.com/assets/1673303/24293608/94f63050-10cd-11e7-83b3-53fd1772187a.png)

Code:
```markdown
An example problem about the interaction between classes: how to lower the 
  <tooltip content=" coupling is the degree of interdependence between software modules; a measure of how closely 
  connected two routines or modules are; the strength of the relationships between modules.">coupling</tooltip>
  between UI and Logic classes?
```

Other things:
* Tip text is better to be left-aligned by default?
* We need a way to indicate the tool-tipped text e.g. a grey dash underline (similar to the one given in #11 )

**Comment by Gisonrg** [comment]
![screen shot 2017-03-24 at 10 37 43 pm](https://cloud.githubusercontent.com/assets/4332224/24299340/e305be80-10e3-11e7-92c6-c065a49be8a1.png)

Left aligned may look strange I think.

I added style for tooltip when used with pure text like the mentioned case (sometimes people want to wrap button/link as well for tooltip, which the underline does not apply).

**Comment by Gisonrg** [comment]
Fix in 1.2.6 release
